File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/executing/EndsWithEvaluator.java
package org.opencds.cqf.cql.engine.elm.executing;


public class EndsWithEvaluator {

    public static Object endsWith(String argument, String suffix) {
        if (argument == null || suffix == null) {
            return null;
        }
        return argument.endsWith(suffix);
    }

}

File: Src/java/engine/src/main/kotlin/org/opencds/cqf/cql/engine/elm/executing/EndsWithEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

object EndsWithEvaluator {
    @JvmStatic
    fun endsWith(argument: String?, suffix: String?): Any? {
        if (argument == null || suffix == null) {
            return null
        }
        return argument.endsWith(suffix)
    }
}
